modules/apigee_edge_debug/apigee_edge_debug.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
```

Comment:
One thing that I have missed - and I see it is also incorrect in apigee_edge.install and also in module - there should be only one `@file` annotation in the file.

Since it has to be fixed in several places, I am not considering this a blocker.

Reply:
Thanks @mxr576, I added this as a new issue: #541